packages/discord.js/src/managers/EntitlementManager.js

```diff
@@ -53,21 +60,49 @@ class EntitlementManager extends CachedManager {
 
   /**
    * Fetches entitlements for this application
-   * @param {FetchEntitlementsOptions} [options={}] Options for fetching the entitlements
+   * @param {EntitlementResolvable|FetchEntitlementOptions|FetchEntitlementsOptions} [options]
+   * Options for fetching the entitlements
    * @returns {Promise<Collection<Snowflake, Entitlement>>}
    */
-  async fetch({ limit, guild, user, skus, excludeEnded, cache = true, before, after } = {}) {
+  async fetch(options) {
+    if (!options) return this._fetchMany(options);
+    const { entitlement, cache, force } = options;
+    const resolvedEntitlement = this.resolveId(entitlement ?? options);
+
+    if (resolvedEntitlement) {
+      return this._fetchSingle({ entitlement: resolvedEntitlement, cache, force });
+    }
+
+    return this._fetchMany(options);
+  }
+
+  async _fetchSingle({ entitlement, cache, force = false }) {
+    if (!force) {
+      const existing = this.cache.get(entitlement);
+
+      if (existing) {
+        return existing;
+      }
+    }
+
+    const data = await this.client.rest.get(Routes.entitlement(this.client.application.id, entitlement));
+    return this._add(data, cache);
+  }
+
+  async _fetchMany({ limit, guild, user, skus, excludeEnded, excludeDelete, cache, before, after } = {}) {
```

Comment:
Shouldn't `cache` default to true as it was previously?

Reply:
It already is defaulted to true in `_add()`. Adding `true` here isn't actually doing anything.